sdks/python/apache_beam/runners/common.py

```diff
@@ -71,6 +70,21 @@ def __init__(self,
                # Preferred alternative to context
                # TODO(robertwb): Remove once all runners are updated.
                state=None):
+    """Initializes a DoFnRunner.
+
+    Args:
+      fn: user DoFn to invoke
+      args: positional side input arguments (static and placeholder), if any
+      kwargs: keyword side input arguments (static and placeholder), if any
+      side_inputs: list of sideinput.SideInputMaps for deferred side inputs
```

Comment:
nit: this should be "sideinputs.SideInputMaps"; period at end of each description.